### PR TITLE
First fluent

### PR DESCRIPTION
**Pull request by jkrol2**

### CODE REVIEW

File: tests/test_default.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+from testinfra.utils.ansible_runner import AnsibleRunner
+
+
+testinfra_hosts = AnsibleRunner('.molecule/ansible_inventory').get_hosts('all')
+
+
+def test_directories(File):
```
Comment (ghost) [inline tests/test_default.py]: Please change used class from `File` to `host` and use newer testinfra fixtures. Example can be found in [ansible-mongo tests](https://github.com/SoInteractive/ansible-mongodb/blob/master/tests/test_default.py)

File: tests/inventory [hunk 2]
```diff
@@ -0,0 +1 @@
+localhost
```
Comment (ghost) [inline tests/inventory]: This file is not necessary

File: templates/td-agent-bit.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+{% for key in input %}
+    {{ key }} {{ input[key] }}
+{% endfor %}
+{% endfor %}
+
+[OUTPUT]
+    Name          forward
+    Match         *
+    Host          "{{ fluentbit_dest_ipaddr }}"
+    Port          24284
```
Comment (ghost) [inline templates/td-agent-bit.conf.j2]: This should be customizable. Also defult port for fluentd transmission is 24224.

File: templates/td-agent-bit.conf.j2 [hunk 4]
```diff
@@ -0,0 +1,45 @@
+[SERVICE]
```
Comment (ghost) [inline templates/td-agent-bit.conf.j2:2]: Start with commented line `# {{ ansible_managed }}` to inform that this file is managed by ansible.

File: tasks/configure.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+---
+
+- name: Copy new td-agent-bit.conf from role's default
+  template:
+    src: td-agent-bit.conf.j2
+    dest: "/etc/td-agent-bit/td-agent-bit.conf"
+  changed_when: false
```
Comment (ghost) [inline tasks/configure.yml]: Why `changed_when` is used with template module?

File: tasks/install.yml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    yum_repository:
+      name: TD_Agent_Bit
+      baseurl: http://packages.fluentbit.io/centos/7
+      gpgcheck: yes
+      gpgkey: http://packages.fluentbit.io/fluentbit.key
+      description: fluent bit  repo
+      enabled: yes
+
+  - name: Install fluentbit
+    yum:
```
Comment (ghost) [inline tasks/install.yml:29]: Please add `update_cache` when installing package from recently added repository.

File: tasks/install.yml [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    apt_key:
+      url: http://packages.fluentbit.io/fluentbit.key
+      state: present
+
+  - name: Add repo
+    apt_repository:
+      repo: deb http://packages.fluentbit.io/ubuntu xenial main
+
+  - name: Install fluentbit
+    apt:
```
Comment (ghost) [inline tasks/install.yml:13]: Please add `update_cache` when installing package from recently added repository.

**Review by ghost — CHANGES_REQUESTED** [review]

File: tasks/install.yml (hunk 7 above)
Comment (jkrol2) [reply tasks/install.yml:13]: update_cache has default value 'yes' in apt_repository module
[http://docs.ansible.com/ansible/latest/apt_repository_module.html](url)

File: tasks/configure.yml (hunk 5 above)
Comment (jkrol2) [reply tasks/configure.yml]: to pass indepotence test

Comment (ghost) [reply tasks/configure.yml]: template module is idempotent by default.

**Comment by ghost** [comment]
@jkrol2 one more thing and this PR is ready to be merged. Please remove binary file `tests/test_default.pyc` and add rule to exclude `*.pyc` files from git repository.

**Review by ghost — APPROVED** [review]

**Pull request merged**